lib/reverse_sentence.rb

```diff
@@ -1,6 +1,48 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
-def reverse_sentence(my_sentence)
-  raise NotImplementedError
+# Time complexity: O(N)
+# Space complexity: O(N)
```

Comment:
Since you're not making another string or array, this is actually O(1) space complexity.